Omise.Examples/Examples/Cards.cs

```diff
@@ -9,7 +9,7 @@ public class Cards : Example
     {
         public async Task List__List()
         {
-            var customerId = "cust_test_566l662pnj240tgz61k";
+            var customerId = TestInfo.CUST_ID;
```

Comment:
@tonydwoodhouse Not sure if we want `TestInfo` class to go into the example code. This will be published onto the docs site na.

Reply:
Maybe we can add a small comment after to show what this looks like

```csharp
var customerId = ExampleInfo.CUST_ID // "cust_test_566l662pnj240tgz61k"
```

Reply:
Amended in latest commit - https://github.com/omise/omise-dotnet/pull/21/commits/86bd95f3e1d576df5a6c14d8538122a6c1af7840